modules/gltf/gltf_document.cpp

```diff
@@ -230,9 +230,11 @@ Error GLTFDocument::_serialize_extensions(Ref<GLTFState> state) const {
 		extensions_required.push_back("KHR_texture_transform");
 	}
 	if (!extensions_used.is_empty()) {
+		extensions_used.sort();
```

Comment:
Need to document that we sort rather than use the order of the gltf extension.

Reply:
Done, added some documentation:

> [...] The final list is sorted alphabetically.